README.md

```diff
@@ -1,63 +1,3280 @@
+investigated for an issue cannot be unduly altered or modified. Support
+investigations are intended as an avenue of exploration and research for
+ensuring that issues are categorised to allow improvements to the product
+surface such that these issues are eliminated.
+
+## 2. Initial Diagnosis
+
+The balenaCloud Dashboard includes the ability to run a set of diagnostics on
+a device, to determine its current condition. This should, in most cases,
+be the first step in attempting to diagnose an issue without having to
+actually access the device via SSH. Ensuring diagnostics and health checks
+are examined first ensures that a support agent has a good idea of the type
+of situation a device is in before SSHing into it, as well as ensuring that
+the information can be accessed later if required (should a device be in a
+catastrophic state). This helps greatly in a support post-mortem should one
+be required.
+
+Currently, diagnosis is only available via the Dashboard.
+
+Let's take a look at the device provisioned earlier that is should now be
+running the code pushed to the DebugApp. Bring up the balenaCloud Dashboard
+page and select 'Diagnostics (Experimental)' from the left-hand panel.
+
+Diagnostics are split into three separate sections: health checks, diagnostics
+and Supervisor state.
+
+### 2.1 Device Health Checks
+
+Select the 'Device Health Checks' tab in the Diagnostics page, and then 'Run
+checks'. A set of health checks will be quickly carried out on-device, and you
+should see the following conditions:
+
+| Check | Status | Notes |
+| --- | --- | --- |
+| check_balenaOS | Succeededsupported | balenaOS 2.x detected |
+| check_under_voltage | Succeeded | No under-voltage events detected |
+| check_memory | Succeeded | 75% memory available |
+| check_container_engine | Succeeded | Container engine balena is running and has not restarted uncleanly
+| check_supervisor | Succeeded | Supervisor is running |
+| check_dns | Succeeded | First DNS server is 127.0.0.2 |
+| check_diskspace | Succeeded | df reports 99% free |
+| check_write_latency | Succeeded | No slow disk writes detected |
+| check_service_restarts | Succeeded | No services are restarting unexpectedly |
+| check_timesync | Succeeded | Time is synchronized |
+
+This shows a healthy device, where there are no obvious faults. That's no fun,
+let's create one!
+
+SSH into your device, via `balena ssh`, using the appropriate UUID. We want to
+SSH into the host OS, as that's where we'll wreak havoc:
+```
+$ balena ssh 7db55ce99e9c135dbc69974a7abbe511
+=============================================================
+    Welcome to balenaOS
+=============================================================
+root@7db55ce:~#
+```
+We're going to do a couple of things that will show up as problems. Something
+you'll often check, and that we'll discuss later, is the state of the balena
+Supervisor (which is responsible for the download, configuration, initialisation
```

Comment:
```suggestion
Supervisor (which is responsible for the download, configuration, initialization
```

Reply:
Done. How come it's not Supervizor, eh? :)